package.json

```diff
@@ -55,8 +57,10 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,md}": [
-      "npm run lint:js"
+    "*.{js,jsx,md,scss}": [
```

Comment:
Given that you created scripts for each type of file, please separate this as well, editing a SCSS file then run prettier for JS files is a waste of time

Reply:
Ah missed this, thanks!